src/runtime/virtcontainers/kata_agent.go

```diff
@@ -178,6 +182,32 @@ func getSandboxPath(id string) string {
 	return filepath.Join(kataHostSharedDir(), id)
 }
 
+// Use in nydus case, guest shared dir is compatible with virtiofsd sharedir
+// nydus images are presented in kataGuestNydusImageDir
+//
+// virtiofs mountpoint: "/run/kata-containers/shared/"
+// kataGuestSharedDir: "/run/kata-containers/shared/containers"
+// kataGuestNydusImageDir: "/run/kata-containers/shared/images"
```

Comment:
Lots of these new functions have these path comments which might be helpful but are also rather distracting. It might be better to move the comments to the unit tests (when there are some :)

Reply:
not too many,  only here, and I think here add comment is ok